apps/storybook-react-native/tailwind-intellisense.config.js

```diff
@@ -4,9 +4,8 @@ const designSystemPreset = require('@metamask/design-system-tailwind-preset');
 module.exports = {
```

Comment:
Move this to a separate PR to improve tailwind linting https://github.com/MetaMask/metamask-design-system/issues/653